### PR TITLE
feat: add french language to the locals

### PR DESCRIPTION
**Pull request by RomainStorai**
*Checklist*: 

- [x] I have read the contribution guidelines. 
- [x] My pull request has a descriptive title
- [x] My pull request targets the main branch of freeCodeCamp.
- [x] I have tested these changes locally on my machine.

Translated auset-en.json to french in order to add the support of the French language.

```diff
+ Created auset-fr.json
```

Related to issue  #5

### CODE REVIEW

File: public/locales/auset-fr.json [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
...ous marquez des points en récupérant des <strong>reliques</strong> perdues lors de conflits passés. Vous devrez <strong>extraire</strong> des reliques des forêts, les déplacer vers un <strong>waystation</strong> du même type, puis <strong>récupérez</strong> ces reliques. Que ces reliques vous appartiennent ou appartiennent à la forêt, cependant, est une autre question.</p><p>Comme < strong>Chevaliers dévots</strong> d'un ordre exilé, vous ignorez le premier coup que vous subissez au combat si vous avez à la fois un guerrier et une relique, en attaque ou en défense. Vous pouvez également déplacer des reliques avec vos guerriers.</ p><p>Vos reliques sont des <strong>trophées prisés</strong>, alors gardez-les en sécurité. Chaque fois qu'un ennemi enlève une relique de quelque manière que ce soit, il marque deux points au lieu d'un et le remet dans n'importe quelle forêt .</p><p>Au fil du temps, vous développerez votre <strong>suite</strong>, trois colonnes de cartes qui vous permettent d'effectuer des actions. Cependant, fouiller et récupérer des reliques mettra votre suite en danger. besoin de planifier à l'avance et de faire preuve de prudence prendre des risques pour réussir.</p>",
```
Comment (EwenDC) [inline public/locales/auset-fr.json:193]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+      "setupTitle": "Configurer le Culte du lézard",
+      "setup": "<i>1.</i> Choisissez une clairière de patrie qui n'est pas adjacente aux patries ennemies.<br/><i>2.</i> Mettez 4 <strong>guerriers</strong> et 1 <strong>jardin</strong> correspondant dans votre pays d'origine. Placez 3 <strong>guerriers</strong> dans les clairières adjacentes aussi uniformément que possible.<br/><i>3.</i> Placez 2 <strong>guerriers< /strong> dans votre boîte Acolytes.<br/><i>4.</i> Remplissez vos pistes Jardins avec des <strong>jardins</strong>, sauf dans l'espace le plus à gauche du jardin sur la carte.<br/> <i>5.</i> Placez votre <strong>marqueur de paria</strong> sur sa face Paria sur n'importe quelle case de la Paria."
```
Comment (EwenDC) [inline public/locales/auset-fr.json:174]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+      "setupTitle": "Configurer le Duché souterrain",
+      "setup": "<i>1.</i> Choisissez une clairière de patrie qui n'est pas adjacente aux patries ennemies.<br/><i>2.</i> Mettez 2 <strong>guerriers</strong> et 1 <strong>tunnel</strong> dans votre pays d'origine. Placez 5 <strong>guerriers</strong> parmi les clairières adjacentes aussi uniformément que possible.<br/><i>3.</i> Placez votre <strong>Panneau de Terrier< /strong> près de la carte. Remplissez vos pistes Bâtiments avec vos <strong>citadelles</strong> et <strong>marchés</strong>. Mettez vos 9 <strong>cartes de ministre</strong> sur votre pile de Ministres Insoumis. Mettez vos 9 <strong>couronnes</strong> sur les cases carrées indiquant les points de victoire."
```
Comment (EwenDC) [inline public/locales/auset-fr.json:181]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 4]
```diff
@@ -0,0 +1,426 @@
...
+      "setupTitle": "Configurer les Dynasties Eyrie",
+      "setup": "<i>1.</i> Choisissez une clairière de patrie sur la clairière du bord de la carte qui a 2+ clairières entre elle et les patries ennemies.<br/><i>2.</i> Mettez 6 <strong>guerriers< /strong> et 1 <strong>perchoir</strong> dans votre patrie.<br/><i>3.</i> Placez n'importe quelle <strong>carte de chef</strong> sur votre emplacement de carte de chef. Gardez l'autre 3 chefs face visible à proximité.<br/><i>4.</i> Placez vos 2 <strong>Vizirs loyaux</strong> sous les emplacements de la colonne Décret comme indiqué sur votre chef actuel.<br/><i> 5.</i> Remplissez vos espaces Perchoirs avec des <strong>perchoirs</strong>, sauf le plus à gauche."
```
Comment (EwenDC) [inline public/locales/auset-fr.json:188]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 5]
```diff
@@ -0,0 +1,426 @@
...
+      "summary": "<p>En tant que marquise de Cat, vous voulez transformer la forêt en une centrale industrielle et militaire. Chaque fois que vous placez un <strong>bâtiment</strong> sur la carte, vous marquez des points. Plus il y a de ce type de bâtiment il y a sur la carte, plus vous marquez de points.</p><p>Pour alimenter cette construction, vous devez développer et protéger une économie interconnectée de <strong>bois</strong>. Construire des infrastructures rend vos virages plus efficaces et vous aide à piocher plus de cartes, alors frappez pour garantir votre droit à l'expansion. Votre armée est légion, vous permettant d'appliquer votre règle d'une poigne de fer, si nécessaire.</p><p>Le siège de votre pouvoir est le < strong>Donjon de la marquise de Cat</strong>, une structure si imposante qu'aucune autre faction ne peut placer de pièces dans sa clairière. De plus, vos <strong>Hôpitaux de campagne</strong> vous aideront à rester dans le combat. de vos guerriers sont retirés, vous pouvez dépenser une carte correspondant à la clairière des guerriers pour les replacer dans votre donjon - tant qu'il est en place. Protégez bien !</p>",
```
Comment (EwenDC) [inline public/locales/auset-fr.json:201]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 6]
```diff
@@ -0,0 +1,426 @@
...
+      "setup": "<i>1.</i> Choisissez 3 clairières de la patrie, chacune adjacente l'une à l'autre.<br/><i>2.</i> Placez 2 <strong>guerriers</strong> dans chacune de vos patries. Placez 1 <strong>guerrier</strong> dans l'autre clairière.<br/><i>3.</i> Placez le <strong>jeton de donjon</strong> dans l'une de vos patries, non adjacente à un ennemi patrie si possible. Mettez 1 <strong>scierie</strong>, <strong>atelier</strong> et <strong>recruteur</strong> sur la carte, chacun dans une patrie différente de la vôtre.<br/>< i>4.</i> Remplissez votre piste Bâtiments avec vos <strong>bâtiments</strong>, à l'exception des espaces les plus à gauche."
```
Comment (EwenDC) [inline public/locales/auset-fr.json:202]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 7]
```diff
@@ -0,0 +1,426 @@
...
...ds</strong> que vous avez économisés et gagné au cours du dernier tour. Cependant, vous devrez également engager et dépenser vos fonds pour développer et protéger votre réseau commercial, en trouvant un équilibre entre les dividendes et la croissance.</p><p>Vous gagnerez des fonds au fur et à mesure d'autres factions achètent vos <strong>services</strong>, que ce soit l'utilisation de vos bateaux fluviaux ou de guerriers mercenaires, ou même l'une des cartes de votre main publique, qui est toutes <strong>à vendre</strong>. Vous pouvez définir les prix de vos services à chaque tour, alors essayez de prévoir de quels services les autres factions auront besoin.</p><p>Construire des comptoirs commerciaux vous aide à vous développer d votre clientèle et vendez plus de services, car une faction peut vous acheter plus de services à mesure qu'elle s'étend dans plus de clairières où vous avez des comptoirs commerciaux. Mais attention, lorsque vos comptoirs sont détruits, ils ne peuvent pas être reconstruits ! Bien que vous ne contrôliez pas beaucoup de clairières, vous pouvez toujours vous déplacer le long des chemins fluviaux puisque vous êtes une faction de <strong>nageurs</strong>.</p>",
```
Comment (EwenDC) [inline public/locales/auset-fr.json:207]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 8]
```diff
@@ -0,0 +1,426 @@
...
...rimez</strong> vos ennemis. À la fin de votre tour, plus vous dirigez de clairières qui n'ont <i>aucun</i> ennemi pièces - pas de guerriers, pas de bâtiments, rien - plus vous marquez de points.</p><p>Pour gagner en puissance et attirer des guerriers, vous devez gagner des objets et les ajouter à votre imposant <strong>Hoard</strong>. Les bottes, les sacs et les pièces augmentent votre <strong>Commandement</strong>, tandis que les marteaux, le thé, les épées et l'arbalète augmentent votre <strong>Prouesse</strong>. Votre <strong>Mépris du Commerce</strong> signifie vous marquez rarement des points en fabriquant des objets, mais vous pouvez voler des objets à vos ennemis avec vos <strong>pilleurs</strong> !</p><p>À la tête des centaines se trouve votre seigneur de guerre, un guerrier-démagogue dont l'humeur <strong>inconstante< /strong> vous donne une capacité pour le tour. Votre seigneur de guerre est obsédé par la thésaurisation, donc plus vous gagnez d'objets, moins vous aurez le choix.</p><p>Déclarez-vous être la vraie voix du Woodland, vous pouvez inciter <strong>mo bs</strong>, qui détruisent les bâtiments et les jetons ennemis, et attaquent les ruines pour leurs objets.</p>",
```
Comment (EwenDC) [inline public/locales/auset-fr.json:222]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 9]
```diff
@@ -0,0 +1,426 @@
...
... sympathie. Bien que vous puissiez ajouter vos propres cartes à vos partisans, vous pouvez inciter vos ennemis à faire de même en provoquant <strong>Outrage</strong>: chaque fois qu'une autre faction retire de la sympathie ou déplace des guerriers dans une clairière sympathique, elle doit ajouter une de ses cartes. propres cartes à vos partisans.</p><p>N'ayez pas peur de mettre en place des points d'étranglement, car vous êtes des experts de la <strong>guerre de guérilla</strong>: lorsque vous défendez au combat, vous utilisez le dé le plus élevé. Placez votre sympathie dans les clairières où le conflit est probable et forcez votre adversaire à vous affronter !</p><p>Les partisans peuvent également être dirigés vers une <strong>révolte</strong> violente, qui détruit <i>tous</i > des pièces de vos ennemis dans une clairière et y place des guerriers et une nouvelle <strong>base</strong>. Les bases augmentent non seulement votre pioche de cartes, mais vous permettent également de former des <strong>officiers</strong>, qui vous donnent des actions gratuites à chaque tour. Protégez bien vos bases ! Si vous perdez une base, vous perdrez également de nombreux partisans et officiers.</p>",
```
Comment (EwenDC) [inline public/locales/auset-fr.json:158]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 10]
```diff
@@ -0,0 +1,426 @@
...
+      "setupTitle": "Configurer le Vagabond",
+      "setup": "<i>1.</i> Placez votre <strong>pion</strong> dans n'importe quelle forêt.<br/><i>2.</i> Mélangez le paquet de quêtes. Piochez 3 <strong>quêtes</strong> à distribuer à proximité.<br/><i>3.</i> Placez le <Sac/>, la <Boot/>, le <Hammer/> et l'<Sword/> <strong>objets de destruction</strong > <i>(marqué d'un\"R\")</i> sous les ruines au hasard, sauf si cela a déjà été fait.<br/><i>4.</i> Mettez le\"{{vagabond}}\" <strong>carte de personnage</strong> dans votre emplacement de carte de personnage. Placez les <InitialStartingItems>, </InitialStartingItems> et <FinalStartingItem/> <strong>éléments de départ</strong> <i>(marqués d'un\"S\")</i> dans votre Cartable et sur leurs pistes correspondantes, si nécessaire.",
```
Comment (EwenDC) [inline public/locales/auset-fr.json:216]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 11]
```diff
@@ -0,0 +1,426 @@
...
...tions. Vous marquez des points de victoire en effectuant les <strong>rituels</strong> appropriés, en révélant les cartes de votre main correspondant aux clairières où vous avez <strong>jardins</strong>, et défausser des cartes pour terminer le rite. Plus vous avez de jardins dans les clairières correspondant à la carte révélée, plus vous marquez de points.</p><p>Votre approche plus douce signifie que vous ne pouvez pas combattre votre ennemis au début; pour ce faire, vous devez radicaliser vos partisans en <strong>acolytes</strong>. Ce faisant, vous pouvez effectuer des <strong>conspirations</strong> dans les clairières du <strong>Paria</strong> , la couleur la plus défaussée au cours du tour précédent.</p><p>Votre <strong>Haine des oiseaux</strong> signifie que vos cartes oiseaux ne sont pas sauvages dans vos rituels. Vos jardins répandent la nouvelle de votre seigneur dragon, attirant des foules de <strong>Pèlerins</strong>, vous dirigez donc une clairière si vous n'y avez qu'un seul jardin. Enfin, vos acolytes obtiennent toujours <st rong>Vengeance</strong> sur les méchants: chaque fois qu'un de vos guerriers est éliminé alors qu'il défendait, vous gagnez un autre acolyte.</p>",
```
Comment (EwenDC) [inline public/locales/auset-fr.json:172]: Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.

File: public/locales/auset-fr.json [hunk 12]
```diff
@@ -0,0 +1,426 @@
...
+      "body": "Chaque joueur place le marqueur de score de la faction de son choix sur la case\"0\" de la piste de score.",
+      "body_vagabondSetUp": "Chaque joueur place le marqueur de score de la faction de son choix sur la case\"0\" de la piste de score. Chaque joueur Vagabond place un marqueur Relation pour chaque faction non Vagabonde en jeu sur la case Indifférent des Relations."
+    },
+    "chooseHand": {
+      "title": "Choisissez les Mains de Départ",
+      "body": "Chaque joueur choisit trois cartes dans sa main à conserver et place les deux autres cartes face cachée sur la pioche commune. Lorsque vous avez terminé, mélangez le jeu partagé."
+    },
+    "setupEnd": {
+      "title": "Commencer à jouer",
+      "body": "La configuration est maintenant terminée. Le jeu commencera avec le Joueur {{count}}. Si vous souhaitez redémarrer le processus de configuration, utilisez le bouton ci-dessous.<p><i>\"Root Automated Setup\" Développé par Ewen Cameron</i><br/><i>Basé sur le jeu de société\"Root\" publié par Leder Games</i><br/><i>Application sous licence CC BY-NC-SA 4.0</i></p>"
```
Comment (EwenDC) [inline public/locales/auset-fr.json:125]: Feel free to add an extra line below my credit to credit yourself for the translation 🙂

e.g. ...`Developed By Ewen Cameron</i><br/><i>French translation by RomainStorai</i><br/><i>Based on the board game`...

**Review by EwenDC — CHANGES_REQUESTED** [review]
Hi, thanks so much for submitting a translation. I am impressed by the speed at which you were able to produce it. There are a couple of issues to resolve before merging, but overall great work and thanks again for submitting!

File: public/locales/auset-fr.json [hunk 13]
```diff
@@ -0,0 +1,426 @@
...
+    "baseExpansionRequired": "Une copie de base de Root est requise pour la configuration",
+    "factionHirelingExcluded": "La faction associée à ce mercenaire est requise pour la mise en place",
+    "hirelingSelected": "Un mercenaire équivalent pour cette faction est déjà en jeu",
+    "landmarkNotEnoughPlayers": "Pas assez de joueurs pour jouer avec ce repère",
+    "lockedFaction": "Impossible de sélectionner la dernière faction tant qu'une faction militante n'a pas été sélectionnée",
+    "mapLandmarkUsed": "Point de repère déjà inclus dans la configuration de la carte",
+    "noDeck": "Aucun deck sélectionné ! Veuillez sélectionner au moins un Deck",
+    "noFaction": "Aucune faction sélectionnée ! Veuillez sélectionner une faction pour jouer",
+    "noLandmark": "Aucun repère sélectionné ! Veuillez sélectionner au moins un point de repère ou définir le nombre de points de repère sur zéro",
+    "noMap": "Aucune carte sélectionnée ! Veuillez sélectionner au moins une carte",
```
[the remark this answers is not in the document]
Comment (RomainStorai) [reply public/locales/auset-fr.json:57]: Yes!

File: public/locales/auset-fr.json [hunk 14]
```diff
@@ -0,0 +1,426 @@
...
+    },
+    "lake": {
+      "name": "Lac",
+      "setupTitle": "Configurer la carte du Lac",
+      "setup": "<i> 1. </i> Prenez le plateau de la carte Lac / Montagne dans la boîte de jeu et placez-le vers le bas avec le côté Lac vers le haut. <br/> <i> 2. </i> Récupérez les 12 marqueurs de costume , retournez-les face cachée et mélangez-les, puis placez-en une sur chaque clairière. <br/> <i> 3. </i> Placez une ruine dans chaque emplacement de la carte marqué d'un \"R \" <i> (quatre au total) </i>. <br/> <i> 4. </i> Placez ces articles sur les emplacements correspondants de la réserve d'articles près du haut de la carte : 2 <Boot/>, 2 <Bag/>, 1 <Arbalète/>, 1 <Marteau/>, 2 <Épée/>, 2 <Thé/>, 2 <Pièce/>. <br/> <i> 5. </i> Placez les deux dés près de la carte.",
+      "landmarkSetup": "Placez la pièce Ferry dans la clairière d'angle qui est également une clairière côtière. Si vous avez une carte de repère \" The Ferry \", sortez-la de la boîte de jeu et placez-la à côté du plateau sur sa face non mise en place."
```
[the remark this answers is not in the document]
Comment (RomainStorai) [reply public/locales/auset-fr.json:365]: Okay, actually I wasn't sure about the french translation of the game (if the card keep the same name as some board game or if it is totally translated). I'm going to tranlate according to the real game.

**Comment by RomainStorai** [comment]
I made the changes you requested. Hope it fits you well!

**Comment by EwenDC** [comment]
Cheers!

**Pull request merged**